test/src/scene/entityVelocityTest.cpp

```diff
@@ -17,13 +18,14 @@ using namespace std::chrono_literals;
  * *) outputs = {RGL_FIELD_ABSOLUTE_VELOCITY_VEC3_F32, RGL_FIELD_RELATIVE_VELOCITY_VEC3_F32, RGL_FIELD_RADIAL_SPEED_F32}
  */
 #ifdef RGL_BUILD_ROS2_EXTENSION
```

Comment:
```suggestion
#if RGL_BUILD_ROS2_EXTENSION
```
It is always defined.